app/src/db/migrations/20231212000000_init.ts

```diff
@@ -41,27 +36,27 @@ export async function up(knex: Knex): Promise<void> {
         knex.schema.createTable('submission', (table) => {
           table.uuid('submissionId').primary();
           table.uuid('assignedToUserId').references('userId').inTable('user').onUpdate('CASCADE').onDelete('CASCADE');
-          table.string('confirmationId', 255);
-          table.string('contactEmail', 255);
-          table.string('contactPhoneNumber', 255);
-          table.string('contactFirstName', 255);
-          table.string('contactLastName', 255);
-          table.string('intakeStatus', 255);
-          table.string('projectName', 255);
-          table.string('queuePriority', 255);
-          table.string('singleFamilyUnits', 255);
-          table.string('streetAddress', 255);
-          table.string('atsClientNumber', 255);
+          table.text('confirmationId');
+          table.text('contactEmail');
+          table.text('contactPhoneNumber');
+          table.text('contactFirstName');
+          table.text('contactLastName');
+          table.text('intakeStatus');
+          table.text('projectName');
+          table.text('queuePriority');
+          table.text('singleFamilyUnits');
```

Comment:
This could feel like a number... though if this is correlated to the housing units column... it's technically behaving as a potential number range. "Should" be fine if we maybe leave as a string, but if we ever need to semantically use this down the line it might be better to consider if we do need to bifurcate this into lower and upper number columns instead...